internal/boxcli/featureflag/feature.go

```diff
@@ -54,10 +54,6 @@ func (f *feature) Enabled() bool {
 	return f.enabled
 }
 
-func (f *feature) Disabled() bool {
```

Comment:
nit: There are also lowercase `disabled()` and `enabled()`. Based on their code, it seems like they mark a feature enabled or disabled. So they "enable" or "disable" a feature flag. 

I think it's better if we change them to that.

Reply:
done!